src/ui/qgstextformatwidgetbase.ui

```diff
@@ -4336,6 +4321,33 @@ font-style: italic;</string>
                            </widget>
                            <widget class="QWidget" name="pagePolygon">
                             <layout class="QGridLayout" name="gridLayout_18">
+                             <item row="1" column="1">
+                              <widget class="QRadioButton" name="radPolygonFree">
+                               <property name="text">
+                                <string>Free (angled)</string>
+                               </property>
+                              </widget>
+                             </item>
+                             <item row="0" column="2">
+                              <widget class="QRadioButton" name="radPolygonOutside">
+                               <property name="text">
+                                <string>Outside polygons</string>
```

Comment:
If possible, I think it'd be better for this to site on a 4th row (i.e. row="3" column="0"). Right now, the minimum width of a three column layout with long strings (offset from centroid + using perimeter (curved) + outside polygons) makes it  too wide to play nicely with the style dock.